zerver/webhooks/dialogflow/doc.md

```diff
@@ -1,17 +1,16 @@
-Get personal message notifications in Zulip for the results of your Dialogflow queries!
+Get personal message notifications in Zulip for the results of your
+Dialogflow queries!
 
 1. {!create-stream.md!}
 
-1. Next, on your {{ settings_html|safe }}, [create a bot](/help/add-a-bot-or-integration) for
-   {{ integration_display_name }}. Make sure that you select
-   **Incoming webhook** as the **Bot type**.
+1. {!create-an-incoming-webhook.md!}
```

Comment:
Should this one be using `create-bot-construct-url` instead? The text below looks very similar to that common text.

Reply:
They are similar, but they generate webhook URLs that require different parameters:
api_key + email:
`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
v.s.
api_key + stream
`{{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream=stream%20name`

So we can't use `{!create-bot-construct-url.md!}` instead of `{!create-an-incoming-webhook.md!}`.

Reply:
OK